### ISSUE TITLE
Launch Control: MIDI port hardcoded

### ISSUE THREAD

**Issue by TimPietrusky** [mmckegg/loop-drop-app#177]
I can't use the *Novation Launch Control* on *Ubuntu*. 

By using the console:

```js
currentProject.context.midiPorts()
["Midi Through Port-0", "Launch Control MIDI 1"]
```

## Problem

[The device name is hard coded as `Launch Control`](https://github.com/mmckegg/loop-drop-app/blob/master/nodes/global-launch-control/object.js#L21).

**Comment by mmckegg** [comment]
fixed in 133658c5b18703e92615d6c083379c5ec3dc28e4, released in v2.14.3! :tada:

**Comment by TimPietrusky** [comment]
:tada: :+1: :balloon:
